sonic_sfp/sfputilbase.py

```diff
@@ -1005,6 +1052,24 @@ def get_transceiver_dom_info_dict(self, port_num):
             transceiver_dom_info_dict['tx3bias'] = dom_channel_monitor_data['data']['TX3Bias']['value']
             transceiver_dom_info_dict['tx4bias'] = dom_channel_monitor_data['data']['TX4Bias']['value']
 
+            # Threshold Data
+            transceiver_dom_info_dict['temphighalarm'] = dom_module_threshold_data['data']['TempHighAlarm']['value']
```

Comment:
are you able to decouple the threshold reading and the voltage/temp/bias reading? since the threshold is constant, we don't want to read them in every iteration for voltage/power/bias reading. This will introduce unnecessary load to CPU since I2C access is time-consuming.

Reply:
Hi,
 I have decoupled it correctly this time. Implemented a new function get_transceiver_dom_threshold_info_dict.

Reply:
This change looks good to me. 

One more thing is that in folder https://github.com/Azure/sonic-platform-common/tree/master/sonic_platform_base/sonic_sfp  we have copies of all the files in https://github.com/Azure/sonic-platform-common/tree/master/sonic_sfp

The intention to keep this copy is that in the near future(could be next release), we will only keep files under ../sonic_platform_base/sonic_sfp   and others will be removed along with plugins.

so please also merge your change to the files under    ../sonic_platform_base/sonic_sfp

Reply:
Hi,
Ported my changes from sonic_sfp to sonic_platform_base/sonic_sfp